Plots/XY/NCL_xy_35.py

```diff
@@ -0,0 +1,168 @@
+                                     ylim=[-1.2, 1.2],
+                                     yticks=np.arange(-1.5, 1.5, 0.5),
+                                     yticklabels=np.arange(-1.5, 1.5, 0.5))
+
+    # Use geocat-viz function to add major and minor ticks on the x and y axes
+    gvutil.add_major_minor_ticks(axesList[ax],
+                                 x_minor_per_major=4,
+                                 y_minor_per_major=5,
+                                 labelsize="small")
+
+# Set standard alpha (transparency) value
+alpha = 0.4
+
+# Plot first graph:
+
+# Plot x, y, and z lines
+line1 = axesList[0].plot(f, x, color='red')
+line2 = axesList[0].plot(f, y, color='limegreen')
+line3 = axesList[0].plot(f, z, color='blue')
+
+
+# Plot second graph:
+
+# Plot x, y, and z lines with the same level of transparency
+line1 = axesList[1].plot(f, x, color='red', alpha=alpha)
+line2 = axesList[1].plot(f, y, color='limegreen', alpha=alpha)
+line3 = axesList[1].plot(f, z, color='blue', alpha=alpha)
+
+
+# Plot third graph:
+
+# Plot x, y, and z lines with varying transparencies
+line1 = axesList[2].plot(f, x, color='red', alpha=0.8)
+line2 = axesList[2].plot(f, y, color='limegreen', alpha=0.4)
+line3 = axesList[2].plot(f, z, color='blue', alpha=0.15)
+
+
+# Plot fourth graph:
+
+# Plot x, y, and z lines and markers with same level of transparency
+# 'ms' is the marker size
+# 'markevery' allows you to place a marker every num steps in the x direction
+# 'mec' is marker edge color
+# 'mfc' is marker face color
+line1 = axesList[3].plot(f, x, 'o', ls='-', color='red', alpha=alpha,
+                         ms=3, markevery=.05, mec='None', mfc='limegreen')
+line2 = axesList[3].plot(f, y, 'o', ls='-', color='limegreen', alpha=alpha,
+                         ms=3, markevery=.05, mec='None', mfc='blue')
+line3 = axesList[3].plot(f, z, 'o', ls='-', color='blue', alpha=alpha,
+                         ms=3, markevery=.05, mec='None', mfc='red')
+
+
+# Plot fifth graph:
+
+# Create RGBA tuples for lines
+lcolor1 = colors.to_rgba('red', alpha=alpha)
+lcolor2 = colors.to_rgba('limegreen', alpha=alpha)
+lcolor3 = colors.to_rgba('blue', alpha=alpha)
+
+# Create RGBA tuples for markers
+mcolor1 = colors.to_rgba('limegreen', alpha=1.0)
+mcolor2 = colors.to_rgba('blue', alpha=0.4)
+mcolor3 = colors.to_rgba('red', alpha=0.15)
+
+# Plot x, y, and z lines and markers- the lines have the same
+# transparency level, but the markers vary in alpha value
+line1 = axesList[4].plot(f, x, 'o', ls='-', color=lcolor1,
+                         ms=3, markevery=.1, mec='None', mfc=mcolor1)
+line2 = axesList[4].plot(f, y, 'o', ls='-', color=lcolor2,
+                         ms=3, markevery=.1, mec='None', mfc=mcolor2)
+line3 = axesList[4].plot(f, z, 'o', ls='-', color=lcolor3,
+                         ms=3, markevery=.1, mec='None', mfc=mcolor3)
+
+
+# Plot sixth graph:
+
+# Create RGBA tuples for lines
+lcolor1 = colors.to_rgba('black', alpha=alpha)
```

Comment:
Why define `lcolor1` `lcolor2` and `lcolor3` if they are all the same value rather than using one variable say `lcolor` for all three lines? Your current method is a bit more readable, but condensing it wouldn't make this that much harder to understand. You can change it or keep what you have. Either is fine. This is just food for thought.

Reply:
I just did that for consistency, I think I will keep it in just because the 5th graph uses that same format of defining line colors as well.